src/index.js

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable no-multiple-empty-lines */
+/* eslint-disable linebreak-style */
+/* eslint-disable semi */
+/* eslint-disable no-console */
+/* eslint-disable indent */
+/* eslint-disable linebreak-style */
+/* eslint-disable no-unused-expressions */
+/* eslint-disable linebreak-style */
```

Comment:
Please remove ESLint disabling comments. It's OK to disable a line or two if there is no way around, but avoid doing this.

Reply:
Done!! Thanks 👍